lib/apiservers/portlayer/swagger.yml

```diff
@@ -440,6 +440,30 @@ paths:
           description: "Error"
           schema:
             $ref: "#/definitions/Error"
+    delete:
+      description: "Remove a container from existence"
+      operationId: ContainerRemove
+      tags: ["containers"]
+      consumes:
+        - application/octet-stream
```

Comment:
Swagger requires a consumes function to function properly. it doesnt really use the octet/stream. we do not stream any bytes through the body.